doc/getting_started/getting_started.rst

```diff
@@ -24,14 +24,14 @@ Install `wstool <http://wiki.ros.org/wstool>`_ : ::
 
 Create A Catkin Workspace and Download MoveIt Source
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-These tutorials rely on the master branch of MoveIt, which requires a build from source.
-You will need to have a `catkin <http://wiki.ros.org/catkin>`_ workspace setup: ::
+Because the version of the tutorials uses the ``master`` branch which is being actively developed, you will most likely need to build all of MoveIt from soruce. You will need to have a `catkin <http://wiki.ros.org/catkin>`_ workspace setup: ::
 
   mkdir -p ~/ws_moveit/src
   cd ~/ws_moveit/src
 
   wstool init .
   wstool merge -t . https://raw.githubusercontent.com/ros-planning/moveit/master/moveit.rosinstall
+  wstool remove  moveit_tutorials  # this is clone'd in the next section
```

Comment:
```suggestion
```
Why do we need this line .? isn't the workspace empty

Reply:
The next section has this line:
git clone https://github.com/ros-planning/moveit_tutorials.git -b master

Which then causes an error. So we could either 
1) Remove it in the next section 
2) Remove it here

I thought this approach was better, because the [older versions of the tutorials](https://ros-planning.github.io/moveit_tutorials/doc/getting_started/getting_started.html) install moveit from debian, so need this line. So keeping it this way keeps things standard. 

However for a beginner, they might think we're crazy and we should remove the second instance of it. Either way is fine with me.